packages/integration-tests/local.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Includes helper functions for replicating Step Function Workflows
+ * locally
+ */
+'use strict';
+
+const path = require('path');
+const fs = require('fs-extra');
+const clone = require('lodash.clonedeep');
+const { template } = require('@cumulus/deployment/lib/message');
+
+/**
+ * Copy cumulus message adapter python folder to each task
+ * in the workflow
+ *
+ * @param {Object} workflow - a test workflow object
+ * @param {string} src - the path to the cumulus message adapter folder
+ * @param {string} cmaFolder - the name of the folder where CMA is copied to
+ * @returns {Promise.<Array>} an array of undefined values
+ */
+function copyCMAToTasks(workflow, src, cmaFolder) {
+  return Promise.all(
+    workflow.steps.map(
+      (step) => fs.copy(src, path.join(step.lambda, cmaFolder))
+    )
+  );
+}
+
+/**
+ * Delete cumulus message adapter from all tasks in the test workflow
+ *
+ * @param {Object} workflow - a test workflow object
+ * @param {string} cmaFolder - the name of the folder where CMA is copied to
+ * @returns {Promise.<Array>} an array of undefined values
+ */
+function deleteCMAFromTasks(workflow, cmaFolder) {
+  return Promise.all(
+    workflow.steps.map(
+      (step) => fs.remove(path.join(step.lambda, cmaFolder))
+    )
+  );
+}
+
+/**
+ * Build a cumulus message for a given workflow
+ *
+ * @param {Object} workflow - a test workflow object
+ * @param {Object} configOverride - a cumulus config override object
+ * @param {Array} cfOutputs - mocked outputs of a CloudFormation template
+ * @returns {Object} the generated cumulus message
+ */
+function messageBuilder(workflow, configOverride, cfOutputs) {
```

Comment:
Non-blocking but here and elsewhere in this PR it could be more maintainable to use an object argument, e.g.
```
function messageBuilder({workflow, configOverride, cfOutputs})
```
this makes it easier to add arguments without being concerned about what order they are in when you make the function call.